services_app/src/main/res/layout/conflict_resolver_field_list.xml

```diff
@@ -10,6 +10,7 @@
         android:id="@+id/conflict_overview_message"
         android:layout_width="match_parent"
         android:lines="3"
+        android:layout_marginBottom="4dp"
```

Comment:
Any reasons for adding a layout_marginBottom and setting it to 4dp?

Reply:
There is not particular reason. Should i remove it?

Reply:
Nope, the reason I asked is to make sure it's consistent everywhere.

why use 4dp instead of 24dp?

Reply:
We can add 4dp to give just a little spacing between text and the button. With 24dp it is a larger spacing. Hence 4dp was used.

Reply:
Great